test/oranif_eunit.erl

```diff
@@ -60,7 +60,7 @@ stmt_execute_test({Safe, _Context, Conn}) ->
 
 %% tests no new APIs, but the following functionalities: drop table, create
 %% table, select count, insert into table
-stmt_execute_test_1({Safe, _Context, Conn}) -> 
+stmtExecute_test_1({Safe, _Context, Conn}) -> 
```

Comment:
Rename : `stmtGetQueryValue_count`

Test lines are grouped incorrecty. It should be:
1. preamble_part_1
1. test_part_1
1. cleanup_part_1
1. _empty_line_
1. preamble_part_2
1. test_part_2
1. cleanup_part_3
...

Move the comments on top each group.

Why counts are floats?
If table deson't exists shouldn't execute already return an error?